packages/workspace/src/schematics/remove/lib/check-targets.ts

```diff
@@ -0,0 +1,44 @@
+import { Tree } from '@angular-devkit/schematics';
+import { updateWorkspaceInTree } from '@nrwl/workspace';
+import { Schema } from '../schema';
+
+/**
+ * Check whether the project to be removed has builders targetted by another project
+ *
+ * Throws an error if the project is in use, unless the `--forceRemove` option is used.
+ *
+ * @param schema The options provided to the schematic
+ */
+export function checkTargets(schema: Schema) {
```

Comment:
Will this falsely find a project using `shared-ui-button` (libs/shared/ui-button) when a user tries to remove `shared-ui-button-awesome` (libs/shared/ui-button-awesome)?

Reply:
It shouldn't do. I'm searching for `shared-ui-button:` not just `shared-ui-button`.